napari/layers/utils/layer_utils.py

```diff
@@ -321,6 +321,57 @@ def get_current_properties(
     return current_properties
 
 
+def _coerce_current_properties_value(
+    value: Union[float, str, int, bool, list, tuple, np.ndarray]
+) -> np.ndarray:
+    """Coerce a value in a current_properties dictionary into the correct type.
+
+    Parameters
+    ----------
+    value : Union[float, str, int, bool, list, tuple, np.ndarray]
+        The value to be coerced.
+
+    Returns
+    -------
+    coerced_value : np.ndarray
+        The value in a 1D numpy array with length 1.
+    """
+    if isinstance(value, (np.ndarray, list, tuple)):
```

Comment:
Optional, but the following is slightly easier to read alternative that also catches 2D+ arrays.

```python
coerced_value = np.atleast_1d(value)
if not (coerced_value.ndim == 1 and coerced_value.shape[0] == 1):
    raise ValueError('current_properties values should have length 1.')
```